config.yaml.example

```diff
@@ -41,10 +42,104 @@ colors:
     selected:
         style: standout
 
-# The tabstop witdth (a tab character moves the indentation to the
-# next multiple of this value)
+
+##  The tabstop witdth (a tab character moves the indentation to the
+##  next multiple of this value)
+##
 tabstop: 8
 
-# The editor to be used by the 'e' key (read from $EDITOR by
-# default, but can be overridden here).
+
+##  The editor to be used by the 'e' key (read from $EDITOR by
+##  default, but can be overridden here).
+##
 # editor: "vi"
+
+
+##  Keybindings
+##
+##  The following commands can be mapped:
+##  * display-pager-only     -- Display the pager full-screen
+##  * display-results-only   -- Display the results list full-screen
+##  * split-focus-pager      -- Split screen, focus on pager
+##  * split-focus-results    -- Split screen, focus on results list
+##  * pager-up               -- Scroll one line up in pager
+##  * pager-down             -- Scroll one line down in pager
+##  * pager-pg-up            -- Scroll one page up in pager
+##  * pager-pg-down          -- Scroll one page down in pager
+##  * pager-scroll-left      -- Scroll eight characters left in pager
+##  * pager-scroll-right     -- Scroll eight characters right in pager
+##  * results-up             -- Move to previous result
+##  * results-down           -- Move to next result
+##  * results-pg-up          -- Move one page up in results list
+##  * results-pg-down        -- Move one page down in results list
+##  * prev-result            -- Move to previous result and update pager
+##  * next-result            -- Move to next result and update pager
+##  * pager-goto-result      -- Update pager with currently selected result
+##  * open-file-in-editor    -- Open file in external editor and jump to
+##                              currently selected result
+##  * exit                   -- Exit the application
+##  * unset                  -- Treat keybinding as if not present, fall back to
+##                           -- alternative binding (used to override keybindings)
+##  A command can be mapped to different keys.
```

Comment:
I would add a »multiple« here, since merely mapping to different keys is the point of configurable keybindings in the first place.